packages/react-renderer-demo/src/pages/mappers/blueprint-component-mapper.md

```diff
@@ -1,7 +1,10 @@
 import Grid from '@material-ui/core/Grid'
 import ListOfContents from '@docs/list-of-contents';
+import ListOfContentsMobile from '@docs/list-of-contents-select';
 
 <Grid container item>
+
+<ListOfContentsMobile file="mappers/blueprint-component-mapper" />
```

Comment:
I don't exactly like that we need a special component.
We can use smart layout and reverse the column/row order on small screens. And make this a part of the normal content component.

It should work since we have a different grid item for the actual content and the content navigation.

Reply:
I would like to make a common layout component in the next PR and refactor all pages to use it. I could include this change in it.

```
import DocPage from '@docs/doc-page'

<DocPage file="bla/bla">
  ...content...
</DocPage>
```

instead of defining `Grid`s and contents in every page.

Reply:
Yep, that is exactly what should happen. Can you add a task to the docs project?

Reply:
Done! 🖖 you can find the task in the issues. I will start working on it right after this is merged.